### DISCUSSION TITLE
Use built-in postgres functions

### DISCUSSION THREAD

**Discussion by nilooy** [PostgREST/postgrest#2404] [Q&A] [answered]
Is there any way we can use internal built-in function/helper functions, 
such as `jsonb_array_elements`
more details https://www.postgresql.org/docs/current/functions-json.html#FUNCTIONS-JSON-PROCESSING-TABLE

or more precisely, 

```sql
select *
from users
cross join jsonb_array_elements(favourite_subjects) as sub
where sub->>'title' = 'math'
```

> ### can I make this SQL query via API ?

## table structure

```sql
create table users(
id bigint generated by default as identity primary key, 
favourite_subjects jsonb);
```

**Comment by steve-chavez** [comment, marked as answer]
You can use any pg function inside a sql function: https://postgrest.org/en/stable/api.html#stored-procedures

**Reply by nilooy** [reply]
I am using the stored procedure currently but thought if there's is any direct way. 
Thanks for the quick response 💯
